gateway/default.conf.template

```diff
@@ -80,7 +111,6 @@ server {
               application/xhtml+xml
               application/xml;
 
-  # all redirects are relative to the gateway
```

Comment:
I'm not quite sure why you're removing all the comments, but this one is, IMO, critical (and actually may not be entirely desirable in this setup) since nginx here has a server name.